andela/ah-backend-stark
# list all users profiles

**What does this PR do?**
This PR lists all authors profile

**Description of Task to be completed**
A logged in user is able to see the profiles of all authors so as to acquire information on them.

**How should this be manually tested**
After checking into the branch ft-list-user-functionality-160584007 , start the server, login then go to the list profiles route http://127.0.0.1:8000/api/profiles/ .You will be able to all registered users. then go to the view profile(for a specific user) route http://127.0.0.1:8000/api/profile/username/ .You will be able to see the profile of that specific user.
To update use the route http://127.0.0.1:8000/api/profile/username/ . and specify your username.

**Screen shot**
<img width="1154" alt="screen shot 2018-10-10 at 13 55 59" src="https://user-images.githubusercontent.com/40117122/46732567-80087a00-cc96-11e8-9c48-9a649d38d530.png">


<img width="1131" alt="screen shot 2018-10-10 at 13 56 42" src="https://user-images.githubusercontent.com/40117122/46732589-8991e200-cc96-11e8-8dd2-e42bd0a61992.png">
